Google.Api.Generator.Tests/ProtoTests/Showcase/Google.Showcase.V1Beta1/IdentityClient.g.cs

```diff
@@ -116,6 +120,16 @@ private IdentitySettings(IdentitySettings existing) : base(existing)
         /// </remarks>
         public gaxgrpc::CallSettings ListUsersSettings { get; set; } = gaxgrpc::CallSettings.FromExpiration(gax::Expiration.None);
 
+        /// <summary>
+        /// The settings to use for the <see cref="gcl::LocationsClient"/> associated with the client.
```

Comment:
nit, clearer and for consistency with some other similar comments.
```suggestion
        /// The settings to use for the <see cref="gcl::LocationsClient"/> associated with this client.
```

Reply:
This is purely generated - we can change ServiceSettingsCodeGenerator at another time, but it shouldn't be part of this PR.